DigitalLearningSolutions.Data.Tests/Services/SectionContentServiceTests.cs

```diff
@@ -331,36 +344,41 @@ public void Get_section_content_should_order_by_tutorial_id_when_shared_order_by
             result.Tutorials.Select(tutorial => tutorial.Id).Should().Equal(expectedTutorialOrder);
         }
 
-        [TestCase(46, 353, 50, 45633, 103)]
-        [TestCase(262288, 22400, 386, 392, 225371)]
-        [TestCase(1, 9850, 101, 170, 284965)]
-        [TestCase(254480, 24224, 101, 1, null)]
-        public void Get_section_content_should_have_same_tutorials_as_stored_procedure(
-            int candidateId,
-            int customisationId,
-            int centreId,
-            int sectionId,
-            int? progressId
-        )
+        [Test]
+        public void Get_section_content_should_sort_using_orderByNumber()
```

Comment:
This seems unrelated to the current change. Has this test just been moved?

Reply:
This was one of the things being checked using the stored procedure `Get_section_content_should_have_same_tutorials_as_stored_procedure`, so I added this when I removed that test 🙂